src/guides/ui-elements/index.md

```diff
@@ -0,0 +1,234 @@
+---
+uid: ui-elements
+---
+
+# User Interface Elements
+
+Abbot supports a limited set of UI elements that can be leveraged in a skill. Unfortunately, these elements only work for Slack and Teams. Discord doesn't yet support UI elements. We may have a workaround for that in the future, but for now, this is the way it is.
```

Comment:
> Unfortunately, these elements only work for Slack and Teams

This feels more factual than unfortunate (it's not the subject of fortune at all). This could be restated as "Discord doesn't support UI elements yet, so these elements only work in Slack and Teams."